### PR TITLE
Update AWS current manifest to v1.1.0

### LINKED ISSUE #2061: Update AWS installation docs for v1.1.0
We want to roll out some doc changes to adopt latest manifests for v1.1.0 release. 

- [x] #2014 
- [x] #2026 
- [x] #2063 
- [x] Updates Installation guidance to use latest v1.1-branch manifests and kfctl

### PR DESCRIPTION
**Pull request by Jeffwan**
resolve #2061

### CODE REVIEW

File: content/en/docs/aws/deploy/uninstall-kubeflow.md [hunk 1]
```diff
@@ -8,8 +8,7 @@ weight = 10
 ## Uninstall Kubeflow
 
 ```
-cd ${KF_DIR}
-kfctl delete -f ${CONFIG_FILE}
+kfctl delete -f kfctl_aws.yaml
```
Comment (Jeffwan) [inline content/en/docs/aws/deploy/uninstall-kubeflow.md]: better to use `-V` to expose more details

**Comment by Jeffwan** [comment]
/cc @joeliedtke @jlewi 

We are good to go. control plane is stable on v1.1. This is the doc update. I only update the AWS tags

**Comment by 8bitmp3** [comment]
Hey @Jeffwan Thanks for reaching out on the v1.1 docs Issue ticket. I can definitely help here. 

1. To speed things up and make sure I'm reviewing the most up-to-date material, I'll fork the repo/branch from `Jeffwan:aws_1.1_change` and introduce a few commits
2. Then, I'll explain them here

cc @jlewi 

Can you please confirm that there following 10 commits need a review:

image

Thanks!

**Comment by PatrickXYS** [comment]
@8bitmp3 Does deploy/preview works for you? https://deploy-preview-2065--competent-brattain-de2d6d.netlify.app/

**Comment by Jeffwan** [comment]
@8bitmp3 Thanks a lot. Feel free to help on the doc review. We basically review technical part by ourselves. @PatrickXYS is from AWS as well and we do peer review.

**Comment by jlewi** [comment]
Lets merge #2072 first so that we don't clobber each other. We've been trying to get the banners added for a while now (look at the original PR). Then @Jeffwan you will want to rebase this PR on that and remove any banners that got added to pages that are being updated in this PR.

@Jeffwan Could please LGTM #2072

**Comment by Jeffwan** [comment]
@jlewi  DONE. Banner change looks good to me

**Comment by Jeffwan** [comment]
@jlewi  @8bitmp3 There's no conflicts and I can skip rebase. Please help approve if that looks good to you.

**Comment by Jeffwan** [comment]
Not sure why it's `Not mergeable`.  `/retest` and have an another try

**Pull request merged**